doc/nrf/protocols/thread/device_types.rst

```diff
@@ -0,0 +1,84 @@
+.. _thread_device_types:
+
+Thread device types
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Thread devices can be either Full Thread Devices (FTD) or Minimal End Devices (MTD).
```

Comment:
should me Minimal Thread Devices (MTD), mb

Reply:
Good catch.